### DISCUSSION TITLE
Do children inherit the parent settings?

### DISCUSSION THREAD

**Discussion by hydroper** [bevyengine/bevy#9049] [Q&A] [answered]
For example, the transform or visibility (opacity), like in Godot?

Looks like this is related: https://github.com/bevyengine/bevy/discussions/3723

**Comment by nicopap** [comment, marked as answer]
yes. It is strictly limited to `Transform` and `Visibility`, and there is no way to disable "inheritance" yet.

**Reply by ickk** [reply]
note that you can bypass visibility inheritance by setting either `Visibility::Visible` or `Visibility::Hidden`; only `Visibility::Inherited` inherits
